<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="MainIndependent" tests="46" time="469.882" ignored="0" failures="4" timestamp="31 Jan 2017 11:48:10 GMT" skipped="0" hostname="EPBYBREW0280" errors="0">
  <testcase name="DetailsDateTest" time="6.690" classname="MainIndependent"/>
  <testcase name="DetailsTagsTest" time="9.067" classname="MainIndependent"/>
  <testcase name="GroupsTestURL" time="21.423" classname="MainIndependent"/>
  <testcase name="DetailsTitleTest" time="7.692" classname="MainIndependent"/>
  <testcase name="DetailsTitleTest" time="15.709" classname="MainIndependent"/>
  <testcase name="AuthorsLinkTest" time="11.635" classname="MainIndependent"/>
  <testcase name="DetailsTagsTest" time="8.948" classname="MainIndependent"/>
  <testcase name="MainPageTitleTest" time="8.109" classname="MainIndependent"/>
  <testcase name="DetailsCameraTest" time="7.210" classname="MainIndependent"/>
  <testcase name="DetailsRightsTest" time="7.138" classname="MainIndependent"/>
  <testcase name="DetailsFollowTest" time="7.680" classname="MainIndependent"/>
  <testcase name="MainPageTitleTest" time="6.757" classname="MainIndependent"/>
  <testcase name="GroupsTestRecommendedGroups" time="4.959" classname="MainIndependent"/>
  <testcase name="GalleriesTestURL" time="8.058" classname="MainIndependent"/>
  <testcase name="MainMenuItemsContTest" time="7.884" classname="MainIndependent"/>
  <testcase name="DetailsNumbersTest" time="7.966" classname="MainIndependent"/>
  <testcase name="GalleriesTestItems" time="4.286" classname="MainIndependent"/>
  <testcase name="MainMenuItemsTest" time="8.057" classname="MainIndependent"/>
  <testcase name="SearchTestUrl" time="6.702" classname="MainIndependent"/>
  <testcase name="AlbumTest" time="35.336" classname="MainIndependent"/>
  <testcase name="ExploreLinkAndPhotoTest" time="25.032" classname="MainIndependent">
    <failure type="java.lang.AssertionError" message="Not each photo passes test! expected [48] but found [14]">
      <![CDATA[java.lang.AssertionError: Not each photo passes test! expected [48] but found [14]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at MainIndependent.ExploreLinkAndPhotoTest(MainIndependent.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- ExploreLinkAndPhotoTest -->
  <testcase name="SubMenuItemsTest" time="12.069" classname="MainIndependent"/>
  <testcase name="AuthorsLinkTest" time="10.944" classname="MainIndependent"/>
  <testcase name="GroupsTestRecommendedGroups" time="4.568" classname="MainIndependent"/>
  <testcase name="NavigateToDetailsTest" time="9.386" classname="MainIndependent"/>
  <testcase name="DetailsFollowTest" time="7.311" classname="MainIndependent"/>
  <testcase name="ExploreLinkAndPhotoTest" time="22.756" classname="MainIndependent">
    <failure type="java.lang.AssertionError" message="Not each photo passes test! expected [48] but found [14]">
      <![CDATA[java.lang.AssertionError: Not each photo passes test! expected [48] but found [14]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at MainIndependent.ExploreLinkAndPhotoTest(MainIndependent.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- ExploreLinkAndPhotoTest -->
  <testcase name="SearchTestPhoto" time="8.630" classname="MainIndependent">
    <failure type="java.lang.AssertionError" message="Your search returned incorrect results, possibly due Firefox!   expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Your search returned incorrect results, possibly due Firefox!   expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at MainIndependent.SearchTestPhoto(MainIndependent.java:253)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- SearchTestPhoto -->
  <testcase name="SearchTestUrl" time="6.010" classname="MainIndependent"/>
  <testcase name="GalleriesTestURL" time="7.833" classname="MainIndependent"/>
  <testcase name="MainMenuItemsContTest" time="7.974" classname="MainIndependent"/>
  <testcase name="NavigateToDetailsTest" time="9.371" classname="MainIndependent"/>
  <testcase name="SearchTestPhoto" time="7.843" classname="MainIndependent">
    <failure type="java.lang.AssertionError" message="Your search returned incorrect results, possibly due Firefox!   expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Your search returned incorrect results, possibly due Firefox!   expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at MainIndependent.SearchTestPhoto(MainIndependent.java:253)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- SearchTestPhoto -->
  <testcase name="GalleriesTestItems" time="3.192" classname="MainIndependent"/>
  <testcase name="DetailsDateTest" time="7.554" classname="MainIndependent"/>
  <testcase name="YouLinkAndNameTest" time="7.417" classname="MainIndependent"/>
  <testcase name="GroupsTestURL" time="13.205" classname="MainIndependent"/>
  <testcase name="DetailsNumbersTest" time="7.332" classname="MainIndependent"/>
  <testcase name="SubMenuItemsTest" time="10.113" classname="MainIndependent"/>
  <testcase name="DetailsAuthorTest" time="7.080" classname="MainIndependent"/>
  <testcase name="AlbumTest" time="34.442" classname="MainIndependent"/>
  <testcase name="DetailsCameraTest" time="7.103" classname="MainIndependent"/>
  <testcase name="DetailsAuthorTest" time="6.830" classname="MainIndependent"/>
  <testcase name="YouLinkAndNameTest" time="8.141" classname="MainIndependent"/>
  <testcase name="DetailsRightsTest" time="7.464" classname="MainIndependent"/>
  <testcase name="MainMenuItemsTest" time="8.976" classname="MainIndependent"/>
</testsuite> <!-- MainIndependent -->
